content/en/docs/installation/supported-releases.md

```diff
@@ -0,0 +1,269 @@
+---
+title: "Supported releases"
+linkTitle: "Supported releases"
+weight: 5
+type: "docs"
+---
+
+<!--
+Inspired by https://istio.io/latest/about/supported-releases/
+-->
+
+This page lists the status, timeline and policy for currently supported
+releases.
```

Comment:
praise: I cannot wait for this to be merged, it looks so useful for people. Thank you for writing it :+1: